cli/internal/env/env.go

```diff
@@ -9,7 +9,55 @@ import (
 	"github.com/vercel/turbo/cli/internal/util"
 )
 
-func getEnvMap() map[string]string {
+// Details contains a map of environment variables broken down by the source
+type Details struct {
```

Comment:
Not blocking: `Details` by itself doesn't tell me much, it's public as well so would make it a _little_ more explicit. 

```suggestion
type BySource struct {
```